NEWS.txt

```diff
@@ -187,6 +188,9 @@ Deprecation
     - The config property `compaction_large_partition_warning_threshold` has been deprecated in favour of the new
       guardrail for partition size. That guardrail is based on the properties `partition_size_warn_threshold` and
       `partition_size_fail_threshold`. The warn threshold has a very similar behaviour to the old config property.
+    - The config property `compaction_tombstone_warning_threshold` has been deprecated in favour of the new guardrail
```

Comment:
Can you amend the description of the JIRA ticket to reflect the correct names. Otherwise I'm getting lost in names :-DD

Reply:
Those examples on the JIRA ticket are quite old and they use the initial nested syntax that was vetoed. I have updated them to the new flat format with the same names that we use on the patch.